packages/components/src/VirtualizedList/VirtualizedList.component.js

```diff
@@ -29,6 +32,16 @@ function VirtualizedList(props) {
 		selectionToggle,
 	});
 
+	if (inProgress) {
+		return (
+			<div className="tc-list-progress">
+				<Status
+					status="inProgress"
+					label={t('LIST_LOADING_LABEL', { defaultValue: 'Loading...' })}
```

Comment:
Avoid using three dots, please change to ellipsis char instead

Reply:
This component will be replaced with CircularProgress